examples/OcclusionSaliency.ipynb

```diff
@@ -143,9 +178,106 @@
       "text/plain": [
-       "<Figure size 432x288 with 3 Axes>"
+       "(3721, (3721, 224, 224))"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Generate the additional imagery for blackbox prediction, and get predictions on those.\n",
+    "pert_images, pert_masks = perturber(test_image)\n",
+    "len(pert_images), pert_masks.shape"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "# This visualization shows the differences in how much regions are masked vs. not masked.\n",
```

Comment:
This line renders weirdly in the Jupyter notebook, it may have been meant as a comment.

Reply:
This is supposed to be a "raw" cell, e.g. it just looks like a raw text block in jupyter notebook. Google colab and github fail to handle raw cells appropriately.